game/assets/localisation/locales/en_US/menus.csv

```diff
@@ -7,7 +7,10 @@ MAINMENU_MULTIPLAYER;Multiplayer
 MAINMENU_OPTIONS;Options
 MAINMENU_CREDITS;Credits
 MAINMENU_EXIT;Exit
-MAINMENU_CHECKSUM;Checksum {checksum}
```

Comment:
You've changed the `en_US` locale but not `en_GB`. This mostly works because the game will fallback to `en_US` when `en_GB` is missing a key, but `en_GB` still has the old `MAINMENU_CHECKSUM;Checksum {checksum}`, for which replacing `short_checksum` fails leaving `Checksum {checksum}` on the screen.